pages/storage_and_backup/object_storage/cold_archive_overview/guide.en-asia.md

```diff
@@ -39,13 +45,15 @@ The service allows you to focus on building and deploying cloud applications whi
 
 The service is fully managed by OVHcloud and accessible through S3 API.
 
-**3 steps operation**
+**5 steps operation**
```

Comment:
As said before, we should indeed finally use a "3-operation process: Store - Archive - Restore" instead, and do the explanation for the optional ones (i'm going to comment the small changes).